packages/playground/src/App.vue

```diff
@@ -235,6 +274,13 @@ function clickHandler({ route, url }: AppRouteItem): void {
 }
 
 $router.beforeEach((to, from, next) => {
+  if (network !== "main") {
+    routes = routes.filter(route => route.title !== "Minting");
+    if (to.path === "/minting") {
+      return;
+    }
+    next();
+  }
```

Comment:
Please don't add check `beforeEach` for network checking instead add a condition for adding minting or not as follow

```
const routes = [
  ...,
  ...(network !== "main"
    ? {}
    : ({
        title: "Minting",
        items: [{ title: "Minting", icon: "mdi-file-document-edit", route: "/minting" }],
      } as any))
  ...,
]
```